lib/backends/sdc/containers.js

```diff
@@ -1507,15 +1507,28 @@ function buildVmPayload(opts, container, callback) {
         });
     }
     if (container.Labels) {
-        Object.keys(container.Labels).forEach(function _labelToTag(key) {
-            if (key === PACKAGE_SELECTION_LABEL) {
+        labelKeys = Object.keys(container.Labels);
+        for (i = 0; i < labelKeys.length; i++) {
+            labelKey = labelKeys[i];
+            if (labelKey === PACKAGE_SELECTION_LABEL) {
                 // PACKAGE_SELECTION_LABEL is a special label. It gets consumed
                 // here and doesn't end up in the user's list. Instead it's used
                 // in getPackage().
-                return;
+                continue;
             }
-            payload.tags[common.LABELTAG_PREFIX + key] = container.Labels[key];
-        });
+            if (triton_tags.isTritonTag(labelKey)) {
+                try {
```

Comment:
I'd propose 2 more options, first the one I'd probably use since I believe it's the idiomatic node way to do this:

```
// Calls callback(err, value)
triton_tags.validateAndNormalizeTritonTagStr(labelKey, container.Labels[labelKey], function _validateCb(err, value) {
    if (err) {
        callback(err);
        return;
    }

    payload.tags[labelKey] = value;
});
```

or if the reason for it being invalid isn't important (like the current code):

```
// separate validation and normalization
if (triton_tags.validTritonTagStr(labelKey, container.Labels[labelKey])) {
    payload.tags[labelKey] = triton_tags.normalizeTritonTagStr(labelKey, container.Labels[labelKey]));
} else {
    callback(new errors.ValidationError('invalid ' + labelKey));
    return;
}
```